DependencyInjection/MonologExtension.php

```diff
@@ -108,6 +111,18 @@ public function load(array $configs, ContainerBuilder $container)
         }
 
         $container->setParameter('monolog.additional_channels', isset($config['channels']) ? $config['channels'] : array());
+
+        if (method_exists($container, 'registerForAutoconfiguration')) {
+            if (interface_exists(ProcessorInterface::class)) {
+                $container->registerForAutoconfiguration(ProcessorInterface::class)
+                    ->addTag('monolog.processor');
+            } else {
+                $container->registerForAutoconfiguration(WebProcessor::class)
+                    ->addTag('monolog.processor');
+            }
+            $container->registerForAutoconfiguration(TokenProcessor::class)
```

Comment:
Why the management of Token is different from WebProcessor?

Reply:
`WebProcessor extends BaseWebProcessor` while `TokenProcessor` doesn't.